### ISSUE TITLE
Run-AlValidation fails, but MS validation passed 

### ISSUE THREAD

**Issue by dkorinets** [microsoft/navcontainerhelper#1520]
I'm trying to run the script 
```
 $validationResults = Run-AlValidation `
    -credential $credential `
    -licenseFile $licenseFileUri `
    -previousApps @( $previousApp ) `
    -apps @( $appFile ) `
    -validateCurrent `
    -countries $supportedCountries `
    -affixes $affixes `
    -supportedCountries $supportedCountries
```

but it fails with the error 

```
Validation errors occured. Error is: [
    "175 errors found in Sana Commerce_Sana Commerce (NA)_10.0.0.18.app on https://bcartifacts.azureedge.net/sandbox/17.1.18256.19767/us:",
    "app.json(9,5): error AL1038: The version number \u0027\u0027 for the property \u0027application\u0027 does not match the expected format: W.X.Y.Z where W, X, Y, and Z represent     
positive integers.",
    ""
]
At C:\Work\Sana\Dev\NavExtension\Scripts\DevOps\ReleaseSanaApp\_RunAlValidation.ps1:25 char:5
+     throw "Validation errors occured. Error is: $($validationResults  ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Validation erro...s.",
    ""
]:String) [], RuntimeException
    + FullyQualifiedErrorId : Validation errors occured. Error is: [
    "175 errors found in Sana Commerce_Sana Commerce (NA)_10.0.0.18.app on https://bcartifacts.azureedge.net/sandbox/17.1.18256.19767/us:",
        "app.json(9,5): error AL1038: The version number \u0027\u0027 for the property \u0027application\u0027 does not match the expected format: W.X.Y.Z where W, X, Y, and Z represent positive integers.",
    ""
]
```

app.json looks fine 
  "name": "Sana Commerce (NA)",
  "publisher": "Sana Commerce",
  "version": "10.0.0.18",
  "platform": "17.0.0.0",

error happens -->
![image](https://user-images.githubusercontent.com/31474116/101467231-b90e0900-394a-11eb-9ab7-ca99cbe0ca6a.png)

**Comment by freddydk** [comment]
The reason for this issue is, that your app.json doesn't have a dependency on "Application", but instead has dependencies on "Base Application" and "System Application" individually.
This is a bug in ContainerHelper, which is fixed in the next preview version.

**Comment by freddydk** [comment]
Fix shipped in 1.0.16
